lib/bullhorn/application.ex

```diff
@@ -9,6 +9,7 @@ defmodule Bullhorn.Application do
 
   def start(_type, _args) do
     children = [
+      {SpandexDatadog.ApiServer, [http: HTTPoison, host: "127.0.0.1"]},
```

Comment:
WDYT of moving the host, or maybe the options here entirely, config and releases.exs?

Reply:
This configuration is static and shouldn't change so I moved it to `config/config.exs` :+1: